code/modules/mob/living/carbon/life.dm

```diff
@@ -53,6 +53,8 @@
 		return
 	if(istype(loc, /obj/machinery/atmospherics/components/unary/cryo_cell))
 		return
+	if(istype(loc, /obj/item/clothing/head/mob_holder)) // Bandaid. It stops mob_holders from breathing, which is sub-optimal.

```

Comment:
This is not a permanent solution. This simply stops them from breathing altogether so they don't die. However that means if they're worn out into space they still don't breathe (oddly enough, affected by pressure still though.)